src/Models/BlogModel.php

```diff
@@ -15,32 +18,36 @@ public function __construct($db)
         $this->db = $db;
     }
 
-    public function GetAllEntries(): bool
+    public function createNewEntry($newBlogPost): bool
     {
-        $query = $this->db->prepare("SELECT `*` FROM `blog-posts` WHERE deleted=0");
-        $query->execute();
-        $Blog = $query->fetchAll();
-        return $Blog;
+        $query = $this->db->prepare('INSERT INTO `blog-posts` (`title`, `author`, `date`, `post`)
+                                        VALUES (:title , :author, :date, :post)');
+        return $query->execute($newBlogPost);
     }
 
-    public function CreateNewEntry($BlogPost): bool
+    public function readAllEntries(): array

```

Comment:
i don't think you can type hint to array here as if the query fails for some reason it will not return and array and break